Doc/whatsnew/3.14.rst

```diff
@@ -75,7 +75,9 @@ New Features
 Other Language Changes
 ======================
 
-
+* :pep:`618`: The :func:`map` function now has an optional ``strict`` flag, used
```

Comment:
I'd rather see ”like `zip` than a reference to PEP-618, which lists this in Rejected ideas.